tools/tests/stepper/test.js

```diff
@@ -39,21 +39,21 @@ describe('Stepper Component', () => {
 
     it('Should have correct selected option', async () => {
         const selected = STEPPER_OPTIONS_SELECTED.find(option => option.selected);
-        await loadStepper(STEPPER_OPTIONS_SELECTED);
+        await loadStepper(STEPPER_OPTIONS_SELECTED, selected.name);
         const stepper = document.querySelector('gameface-stepper');
         assert.equal(stepper.value, selected.name);
     });
 
     it('Should have correct selected index', async () => {
-        const selected = STEPPER_OPTIONS_SELECTED.findIndex(option => option.selected);
-        await loadStepper(STEPPER_OPTIONS_SELECTED);
+        const selectedIndex = STEPPER_OPTIONS_SELECTED.findIndex(option => option.selected);
+        await loadStepper(STEPPER_OPTIONS_SELECTED, STEPPER_OPTIONS_SELECTED[selectedIndex].name);
         const stepper = document.querySelector('gameface-stepper');
-        assert.equal(stepper.selectedIndex, selected);
+        assert.equal(stepper.selectedIndex, selectedIndex);
     });
 
     it('Should render correct value', async () => {
         const selected = STEPPER_OPTIONS_SELECTED.find(option => option.selected);
-        await loadStepper(STEPPER_OPTIONS_SELECTED);
+        await loadStepper(STEPPER_OPTIONS_SELECTED, selected.name);

```

Comment:
How would the second argument be taken into account in `loadStepper` function?
![image](https://github.com/CoherentLabs/GameUIComponents/assets/21058220/e10784d8-0251-4ea9-a9b0-32ec7eb73292)

Reply:
Are you sure you have checked the differences from all the commits and not a single commit change?
The real source of this is the following
```
const loadStepper = (options, value = '') => {
    const optionElements = options.map((option) => {
```

Reply:
All good!
Interesting, my fault or GitHub showed me something stale cuz this is a screenshot from here :D